tests/scripts/analyze_outcomes.py

```diff
@@ -20,18 +20,25 @@ def __init__(self):
         self.error_count = 0
         self.warning_count = 0
 
+    # Private method
     @staticmethod
-    def log(fmt, *args, **kwargs):
+    def __log(fmt, *args, **kwargs):
         sys.stderr.write((fmt + '\n').format(*args, **kwargs))
 
     def error(self, fmt, *args, **kwargs):
-        self.log('Error: ' + fmt, *args, **kwargs)
+        self.__log('Error: ' + fmt, *args, **kwargs)
         self.error_count += 1
 
     def warning(self, fmt, *args, **kwargs):
-        self.log('Warning: ' + fmt, *args, **kwargs)
+        self.__log('Warning: ' + fmt, *args, **kwargs)
         self.warning_count += 1
 
+    # This is a static method because we don't need to track any data about
+    # the number of times it is called
+    @staticmethod
+    def info(fmt, *args, **kwargs):
```

Comment:
Why do we need `info` function? we could not modify this class and call `Results.log()` instead `print`?